action/rlp_tx.go

```diff
@@ -88,5 +88,13 @@ func DecodeRawTx(rawData string, chainID uint32) (tx *types.Transaction, sig []b
 // NewEthSigner returns the proper signer for Eth-compatible tx
 func NewEthSigner(txType iotextypes.Encoding, chainID uint32) (types.Signer, error) {
```

Comment:
if the signer of iotex format could be returned, the name of this function should be updated to `NewSigner`